app/models/account_user.rb

```diff
@@ -0,0 +1,3 @@
+class AccountUser < ActiveRecord::Base
+  enum role: [:user, :admin]
```

Comment:
does this work without the explicit `belongs_to :user` and `belongs_to :account`?